MapboxNavigation/PassiveLocationManager.swift

```diff
@@ -74,6 +84,16 @@ open class PassiveLocationManager: NSObject, MGLLocationManager {
 }
 
 extension PassiveLocationManager: PassiveLocationDataSourceDelegate {
+    @available(iOS 14.0, *)
+    public func passiveLocationDataSourceDidChangeAuthorization(_ dataSource: PassiveLocationDataSource) {
+        // MGLLocationManager.locationManagerDidChangeAuthorization(_:) was introduced in Mapbox Maps SDK for iOS v6.2.0, but v6.0.0–v6.1.x are still supported.
+        let locationManagerDidChangeAuthorization = Selector(("locationManagerDidChangeAuthorization:" as NSString) as String)
+        guard let delegate = delegate, delegate.responds(to: locationManagerDidChangeAuthorization) else {
+            return
+        }
+        delegate.perform(locationManagerDidChangeAuthorization, with: self)
```

Comment:
Ordinarily, I’d just write `delegate?.locationManagerDidChangeAuthorization(self)` and be done with it. However, MapboxNavigation is still compatible with map SDK v6.0.0–v6.1._x_, so this call requires some runtime reflection. We also have to add some redundant casting of the selector to suppress a warning from the compiler urging us to use the static `#selector` syntax that would ordinarily be safer. This dynamic selector runs the risk that a future map SDK 6._x_ might expect something other than an MGLLocationManager as the first argument, but that seems unlikely given the intention for MGLLocationManagerDelegate to mimic CLLocationManagerDelegate.